FirebaseMessaging/Sources/FIRMessagingConstants.h

```diff
@@ -59,6 +59,8 @@ FOUNDATION_EXPORT NSString *const kFIRMessaging_CMD_RST;
 FOUNDATION_EXPORT NSString *const kFIRMessagingAllScopeIdentifier;
 /// The scope used to save the IID "*" scope token.
 FOUNDATION_EXPORT NSString *const kFIRMessagingDefaultTokenScope;
+/// Blacklisted "fiam" token scope.
```

Comment:
Use "Denylist" instead - here and throughout.

Reply:
Updated comments.